app/views/admin/api_docs/base/swagger.html.erb

```diff
@@ -37,18 +37,80 @@
       }
     }
 
+    var AUTH_METHOD_USER_KEY = 1;
```

Comment:
Personally I am not a big fan of populating the global scope with constants or stuff that we won't need there. So I would put all this new stuff inside the anonymous function before (the one currently in line 96)